File: competitions/semcomp2021/K.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    unsigned long long int n;
    unsigned long long int l,r,k,qntdQM,pQM, ini;
    cin>>n;
    while(n>0){

    n-=1;
    cin>>l>>r>>k;
    qntdQM = 0;
    if(k>r){
        cout<<0<<endl;
    }
    else if(l>=k){

        pQM = l%k;
        if(pQM != 0){
            ini = (l-pQM) + k;
        }
        else{
            ini = l;
        }
        qntdQM = 1+trunc((r-ini)/k);
        cout<<qntdQM<<endl;
    }
    else if(k>l){

        ini = k;
        qntdQM = trunc(r/k);
        cout<<qntdQM<<endl;
    }
    }
}

File: competitions/semcomp2021/K.py
from sys import stdin, stdout
n = int(input())
 
# input the array

while(n>0):
    n-=1
    l,r,k = map(int,input().split())
    qntdQM = 0
    if(k>r):
        stdout.write(str(0)+'\n')
    elif(l>=k):
        pQM = l%k
        if pQM != 0:
            ini = (l-pQM) + k
        else:
            ini = l
        qntdQM = 1+((r-ini)//k)
        stdout.write(str(qntdQM)+'\n')
    elif(k>l):
        ini = k
        qntdQM = (r//k)
        stdout.write(str(qntdQM)+'\n')